src/utils/Common.ts

```diff
@@ -0,0 +1,27 @@
+type Mappable = {
+  [key: string]: any;
+};
+
+export const getPropertyFromPath = (object: Mappable, path: string | string[]): any => {
+    if (!object) {
```

Comment:
I'd also not worry about this check. TS won't allow a potential undefined/null value to be passed and we should be accounting for it outside of this method.

It wouldn't be a good idea to pass around potential null/undefined values in the first place, so we should be handling that prior to making any calls against the object anyway.